packages/frontpage/app/(app)/_components/post-card.tsx

```diff
@@ -147,7 +147,10 @@ export async function PostCard({
 export async function deletePostAction(rkey: string) {
   "use server";
   await ensureUser();
-  await deletePost(rkey);
+  const atproto = atprotoClient();
+  await atproto.fyi.unravel.frontpage.post.delete({
+    rkey,
+  });
```

Comment:
This is essentialy untyped because of a bug in the lex-cli generated code. See https://github.com/bluesky-social/atproto/issues/2952

I'll probably leave this in draft until a solution is found for that.